### DISCUSSION TITLE
Error when installing saleor which docker

### DISCUSSION THREAD

**Discussion by thanhnguyen2603** [saleor/saleor#11626] [Q&A]
I encountered this problem while installing saleor on my mac, i am new to python so still many things are not clear, can someone tell me where i did wrong i have been struggling with this error for 4 hours.
`> [saleor-platform-worker stage-1 10/10] RUN SECRET_KEY=dummy STATIC_URL=/static/ python3 manage.py collectstatic --no-input:
#0 0.747 /bin/sh: 1: python3: Exec format error`
`failed to solve: executor failed running [/bin/sh -c SECRET_KEY=dummy STATIC_URL=${STATIC_URL} python3 manage.py collectstatic --no-input]: exit code: 126
mac@i-fucking-love-uuuu saleor-platform %`

**Comment by peelar** [comment]
Hello @thanhnguyen2603 👋 

Can you write down the steps to reproduce? At what point do you see this error?
